appendix-basic65.tex

```diff
@@ -1536,7 +1654,7 @@ \subsection{CHANGE}
       \item \megakey{*} \specialkey{RETURN} replace the current and all following matches
       \item \specialkey{RETURN} exit the command, and don't replace the current match
     \end{itemize}
-\item [Remarks:] Any un-shifted character that is not part of the string can be
+\item [Remarks:] Any punctuation character that is not part of the string can be
```

Comment:
Bit shifter had previously said "Any un-shifted" character.

The older text appears more accurate, as I tried typing `FIND ZPRINTZ`, and it successfully found all PRINT calls within my code.

Reply:
Agreed that I have narrowed the description inaccurately, but "un-shifted" doesn't work. I think he meant "any character in the un-shifted character set" which I don't want to explain anywhere near here. You hold down Shift to type many of these characters, so "un-shifted" is confusing. Suggestions welcome.

Reply:
Hmm, yeah, it's tricky to do it without being overly wordy eh. Maybe:

"Almost any character(footnote) can be used in place of '/')."

Then the footnote could elaborate that we mean that it is any character formed by a single keypress, without the need for holding down the SHIFT or MEGA keys.

Reply:
"Almost any" looks fine, even without needing to be a complete explanation. Another try: https://github.com/MEGA65/mega65-user-guide/commit/7b1004c3b745327babd5b093643e9834549a9963